src/js/intl/strings.js

```diff
@@ -1884,7 +1890,7 @@ exports.strings = {
   'goal-only-main': {
     '__desc__': 'the helper message for the window that shows the goal tree when the goal will only be compared using the main branch',
     'en_US': '<span class="fwber">Note:</span> Only the main branch will be checked in this level. The other branches are simply for reference (shown as dashed labels below). As always, you can hide this dialog with "hide goal"',
-    'fr_FR': '<span class="fwber">Note:</span> Dans ce niveau on ne doit faire de git checkout que sur la branche main. Les autres branches représentées ici (entourées d\'un cadre en tirets) doivent être utilisées uniquement comme références. Comme toujours, vous pouvez cacher cette fenêtre avec "Cacher les objectifs"',
+    'fr_FR': '<span class="fwber">Note:</span> Dans ce niveau on ne doit faire de git checkout que sur la branche main. Les autres branches représentées ici (entourées d\'un cadre en tirets) doivent être utilisées uniquement comme références. Comme toujours, vous pouvez cacher cette fenêtre avec "hide goal"',
```

Comment:
"Cacher les objectifs" refers to the label of the button (and is correct), but here we must refer to the command to type